lib/tasks/grda_warehouse.rake

```diff
@@ -342,6 +342,8 @@ namespace :grda_warehouse do
     AppResourceMonitor::CollectStatsJob.perform_later if stats_collector.should_enqueue?
 
     BuildTranslationCacheJob.perform_later
+
+    PgheroCollectStatsJob.perform_later(clean: DateTime.current.hour == 5) if !Rails.env.production? && PgHero.query_stats_enabled?
```

Comment:
collecting stats on the hour seem okay? I don't know how frequently we should run this

Reply:
Do you know the volume of data involved? If it's storing information about all tables or all query shapes, it might be big and slow. I imagine it won't be intensive on the database, so hourly is probably fine I'd guess.

Reply:
Good point, I believe the max number of queries tracked by pg_stat_statements is 5000 by default (`SHOW pg_stat_statements.max`). Of course, the sql statements themselves could be quite long. My intuition is this shouldn't be a performance issue. We could keep an eye on staging after this is deployed and tune or disable stats collection if it seems to be a problem